integration_test/perf_driver.dart

```diff
@@ -0,0 +1,24 @@
+// This file is derived from BSD licensed example code in
+// Flutter documentation about integration tests profiling:
+//   https://docs.flutter.dev/cookbook/testing/integration/profiling#3-save-the-results-to-disk
+//
+// This integration driver configures output of timeline data
+// and a summary thereof from integration tests. See
+// docs/integration_tests.md for background.
+
+import 'package:flutter_driver/flutter_driver.dart' as driver;
+import 'package:integration_test/integration_test_driver.dart';
+
+Future<void> main() {
+  return integrationDriver(
+    responseDataCallback: (data) async {
+      if (data != null) {
```

Comment:
nit: let's de-indent by using an early return:
```suggestion
      if (data == null) return;
```